tests/python/relay/test_vm.py

```diff
@@ -1112,6 +1112,20 @@ def test_multi_targets():
     tvm.testing.assert_allclose(actual_result.numpy(), expected_result)
 
 
+def test_dialect_rewriter():
+    """Tests dialect rewriter for tuple node."""
+
+    x = relay.var("x", shape=(3,), dtype="int32")
+    y = relay.take(x, relay.const(0))
+    z = relay.const(1)
+
+    f = relay.Function([x], relay.stack((z, y), axis=0))
+    mod = IRModule.from_expr(f)
+
+    compiler = VMCompiler()
+    compiler.optimize(mod, "llvm")
```

Comment:
nit: thanks for the unit test. Since there's no assertion, please comment that this tests for an ICHECK failure for ill-formed IL with let-bound constants. The test name can similarly describe what is being testing.